### DISCUSSION TITLE
20220904 Update: Networks page - code error (Solved)

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1013] [Fixed Issues] [answered]
Just updated to 20220904 and now get this error on the Networks page:

Errors: Bad control character in string literal in JSON at position 244
{

  "page"       : "networks"
, "activebt"   : true
, "activeeth"  : false
, "activewlan" : true
, "camilladsp" : false
, "ipeth"      : ""
, "ipwlan"     : "192.168.86.35"
, "listbt"     : [ {
  "mac"       : "many"
, "name"      : "arguments▮[0m"        <-- Odd characters here
, "connected" : false
, "type"      : ""
} ]
, "listeth"    : false
, "listwl"     : [ {
	  "dbm"      : -47
	, "gateway"  : "192.168.86.1"
	, "ip"       : "192.168.86.35"
	, "ssid"     : "CMHomeNET"
	} ]
, "hostapd"    : false
, "hostname"   : "rAudio-1"
}

Chris

**Comment by rern** [comment]
The update has missed to upgrade `bluez`. Please try today's update.

**Comment by ceejayemm** [comment]
@rern 

Applied today's update. Bluez seems to have been updated but still same error message.

Chris

**Comment by rern** [comment]
- Post the output of: `bluetoothctl devices Paired`
- If there's an error: `pacman -S bluez-utils`

**Comment by ceejayemm** [comment]
@rern 

Command you gave errors with:

rAudio-1:~  #  bluetoothctl devices Paired
Too many arguments

Revised to:

rAudio-1:~  #  bluetoothctl devices       
Device 08:EB:ED:7F:C8:B7 SoundCore 2

I think you might have meant:

rAudio-1:~  #  bluetoothctl paired-devices       
Device 08:EB:ED:7F:C8:B7 SoundCore 2

both commands give me what I would expect. I only use the SoundCode2 BT speakers with this system, no other output device.

Regards.

Chris

**Reply by rern** [reply]
> ...
> * If there's an error: `pacman -S bluez-utils`

**Comment by ceejayemm** [comment, marked as answer]
@rern 

Sorry, misunderstood what you wanted me to do. I have run the additional 'pacman' command and the original bluetoothctl command now returns:

rAudio-1:~  #  bluetoothctl devices Paired
Device 08:EB:ED:7F:C8:B7 SoundCore 2

The Network page also now works correctly with no errors.

Many thanks

Chris
